lms/services/group_info.py

```diff
@@ -18,6 +18,7 @@ class GroupInfoService:
     GROUPING_TYPES = {
         "course": "course_group",
         "canvas_section": "section_group",
+        "canvas_group": "canvas_group_group",
```

Comment:
This is not a great name but the other two already are on the DB and using _grouping on the last one might be more confusing.